script/lib/utils.js

```diff
@@ -6,6 +6,8 @@ const ELECTRON_DIR = path.resolve(__dirname, '..', '..');
 const SRC_DIR = path.resolve(ELECTRON_DIR, '..');
 
 const RELEASE_BRANCH_PATTERN = /(\d)+-(?:(?:[0-9]+-x$)|(?:x+-y$))/;
+const MAIN_BRANCH_PATTERN = /^(main|master)$/;
```

Comment:
For a list of two things, I'd prefer if this was literally an array and we did a `.includes`.  Can we also go about this strategically and map any change we make to support with with a `// TODO(main-migration): Remove once main branch is renamed` so that we can really easily _undo_ these once we're ready

Reply:
I was planning to make separate PRs for each instance of this in order to make it easier to review, but I can collect them all into one place for ease of reverting if that's preferable?

I started with an array but switched to a regex because it matches the other code here. I think it's easier to read if it's all the same pattern.

Reply:
> I was planning to make separate PRs for each instance of this in order to make it easier to review, but I can collect them all into one place for ease of reverting if that's preferable?

The revert isn't going to be a `git revert`.  I'm down to do this piecemeal (I think it's the safest way to do this) I just want each instance we do to be mapped with a comment / flag somehow so that when we need to remove the "support both branches" part of our code it's very easy to grep for everything we did.

> I started with an array but switched to a regex because it matches the other code here

The other regexps in this file are regexps for patterns not for a known set of allowed values, e.g. detecting release branches rather than saying "is the branch one of these two possibilities".  Throwing a regexp at a "is string A or B" seems over the top but I don't care infinitely 😆

Reply:
I'm +0 on this, you're -0, i think that cancels out so let's leave it as written :)

Reply:
Ah, I misunderstood re: the note -- you meant to come back in future and change it so we only support `main` and drop support for `master`. Not so much "undo" as "finish doing".